app.py

```diff
@@ -230,6 +230,19 @@ def run(self):
                 self.label_studio.run(action="update_tasks", videos=self.extract_ui.st_frame_files)
                 self.extract_ui.run_script_zipped_frames = False
 
+        if self.extract_ui.proj_dir and self.extract_ui.run_script_video_model:
+            self.extract_ui.run(
+                action="extract_frames_using_model",
+                video_files=self.extract_ui.st_video_files,  # add arg for run caching purposes
+            )
```

Comment:
do we need `model_dir=self.extract_ui.model_dir` as an additional arg? this is the case in `labeling_app.py`